src/styles/blocks/product-catalog.scss

```diff
@@ -0,0 +1,29 @@
+.product-catalog {
+  padding-block: 50px;
+  padding-inline: 40px;
+
+  display: grid;
+  grid-template-columns: repeat(1, minmax(200px, 200px));
+  grid-auto-rows: 1fr;
+  gap: 46px 48px;
+  justify-content: center;
+  justify-items: center;
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, minmax(200px, 200px));
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      grid-template-columns: repeat(3, minmax(200px, 200px));
```

Comment:
Replace `minmax(200px, 200px)` with `200px` for consistency and simplicity, as mentioned in the previous comments.